src/util/getBreakpoints.js

```diff
@@ -10,7 +10,15 @@ function getBreakpoints(store) {
     let responsiveStateKey
     // if the redux state root is an Immutable.js Iterable
     if (storeState['@@__IMMUTABLE_ITERABLE__@@'] === true) {
-        responsiveStateKey = storeState.findKey(stateBranch => stateBranch._responsiveState)
+        responsiveStateKey = storeState.findKey(stateBranch => {
+            //  handle responsive state as Immutable Map
```

Comment:
You can actually just return `stateBranch.get('_responsiveState')` here, that will work whether it's a Map or a Record (no need to do a type check).

Reply:
I tried that and it didn't work in certain cases.  I had an example reducer that was neither an Immutable Map or Record and therefore had no .get() method.  I found it easiest and safest just to test for Immutable Map (which should be main data structure responsive state should be in if using immutableJS) and use .get() and fall back to standard object property getter. 

I did entertain just checking for the presence of a .get() method on stateBranch and using it if it was there else falling back ... this may be a better approach.

Reply:
@bguggie do you mind adding a test for both cases just so we can make sure that we have all of ours bases covered?

Reply:
@AlecAivazis Somehow I knew that request was coming.  Yes, I can

Reply:
@bguggie Weird, all Immutable types should have a `.get` method - which type wasn't working?